src/analysis/modals/CromwellModal.js

```diff
@@ -78,11 +79,16 @@ export const CromwellModalBase = withDisplayName('CromwellModal')(
         : h(
             ButtonPrimary,
             {
-              href: app?.proxyUrls['cbas-ui'],
+              href: cloudProvider === cloudProviderTypes.GCP ? app?.proxyUrls['cromwell-service'] : '',
```

Comment:
This `href` didn't have a check against `cloudProvider` before, and I don't know if this is strictly necessary. But I added the check because these two buttons should behave the same way. There were enough inconsistencies in the context bar button behaviors that it seemed likely that this was an oversight.

I judged this change to be worthwhile, in the spirit of "leave things better than when you found them". The absolute safest alternative would be to simply remove the `href`.